src/routes/tokens/components/ApiToken.tsx

```diff
@@ -0,0 +1,178 @@
+import { useRef, useState } from 'react';
+
+import {
+    Button,
+    Text,
+    VStack,
+    HStack,
+    Center,
+    Box,
+    Spacer,
+    AlertDialog,
+    AlertDialogBody,
+    AlertDialogContent,
+    AlertDialogFooter,
+    AlertDialogHeader,
+    AlertDialogOverlay,
+    useDisclosure,
+} from '@chakra-ui/react';
+
+import { deleteToken } from '@/api/BearerTokenApi';
+import { useToast } from '@/hooks/useToast';
+import { shortFormatDate } from '@/libs/utils';
+import { BearerTokenT, NewBearerTokenT } from '@/types/BearerTokenTypes';
+
+const ApiToken = ({ token }: { token: BearerTokenT | NewBearerTokenT }) => {
+    const apiToken = useState<BearerTokenT | NewBearerTokenT>(token)[0];
+    const [isRevoked, setIsRevoked] = useState<boolean>(false);
+    const [isHovered, setIsHovered] = useState<boolean>(false);
+    const toast = useToast();
+    const {
+        isOpen: isConfirmOpen,
+        onOpen: onConfirmOpen,
+        onClose: onConfirmClose,
+    } = useDisclosure();
+    const cancelRef = useRef<HTMLButtonElement>(null);
+
+    const revokeToken = async () => {
+        try {
+            await deleteToken(token.id);
+            setIsRevoked(true);
+            toast({
+                title: 'Token deleted',
+                descriptionComponent: 'The old token is no longer valid.',
```

Comment:
Maybe we could make this error message a bit clearer (it looks like it's from the previous version where you could only have a single token at a time).